lib/beaker-pe/install/pe_utils.rb

```diff
@@ -448,6 +455,35 @@ def deploy_frictionless_to_master(host)
           end
         end
 
+        # Check for availability of required network resources
+        # @param [Array<Host>] hosts
+        # @param [Array<String>] network_resources, defaults to NETWORK_RESOURCES
+        #
+        # @example
+        #   verify_network_resources(hosts, network_resources)
+        #
+        # @return nil
+        #
+        # @api private
+        #
+        def verify_network_resources(hosts, network_resources = NETWORK_RESOURCES)
+          logger.notify("Checking the availability of network resources.")
+          hosts.each do |host|
+            network_resources.each do |resource|
+              on host, "curl -s #{resource} > /dev/null", :acceptable_exit_codes => (0...256)
```

Comment:
instead of `:acceptable_exit_codes => (0...256)`

you can do

```
:accept_all_exit_codes => true
```


https://github.com/puppetlabs/beaker/blob/696b223898cf6daf836c33b88e6355d3777238f8/lib/beaker/dsl/helpers/host_helpers.rb#L15

Reply:
Alternatively, we could be more specific here, since ping should only return exit codes 0, 1, and 2. I think if it returns outside that range, something has gone horribly wrong.